docs/configuration.asciidoc

```diff
@@ -616,6 +616,26 @@ IMPORTANT: Use of `ServerUrls` is deprecated. Use `ServerUrl`.
 | `http://localhost:8200` | String
 |============
 
+[float]
+[[use-windows-credentials]]
```

Comment:
This should be `config-use-windows-credentials` to match the other config anchors and the link in All options summary.

```suggestion
[[config-use-windows-credentials]]
```